bin/upload_repo_deb

```diff
@@ -6,60 +6,50 @@ echo "Uploading $DISTRO"
 echo "Collected debian packages in build/"
 ls -1 build/
 
-copy_to() {
-  echo -e "Copy $1 to /mnt/repo_storage/manticoresearch_$DISTRO$SUFFIX/dists/$2";
-  cp "$1" "/mnt/repo_storage/manticoresearch_$DISTRO$SUFFIX/dists/$2" && echo -e "Success"
-  echo
-}
-
-# bundleamd=0
-# bundlearm=0
+declare -A archs
 
 for f in build/*deb; do
-  # VER="$(echo "$f" | cut -d_ -f2)"
+
+  FILE_NAME="$(basename $f)"
+
+  VER=$(echo $f | cut -d_ -f2)
   ARCH=$(echo "$f" | cut -d_ -f3 | cut -d. -f1)
   echo "Arch: $ARCH"
-  declare -A archs
+
   if [ -f "$f" ]; then
     if [ -z "${IS_RELEASE_DIGIT}" ]; then
       IS_RELEASE_DIGIT=$(echo "$f" | cut -d. -f3 | cut -d- -f1)
-      if [[ $((IS_RELEASE_DIGIT % 2)) -ne 0 ]]; then
-        SUFFIX="_dev"
+      if [[ $(($IS_RELEASE_DIGIT % 2)) -ne 0 ]]; then
+          TARGET="dev"
+          SUFFIX="_dev"
+        else
+          TARGET="release"
+          SUFFIX=""
       fi
     fi
 
-    ./bin/sign_deb "$GPG_SECRET" "$f"
-
     if [[ $ARCH == "amd64" ]]; then
       archs[amd]=1
-      copy_to "$f" "$DISTRO/main/binary-amd64/"
-      # bundleamd=1
+      /usr/bin/docker exec -i repo-generator /generator.sh --distro $DISTRO --architecture amd --target $TARGET --name $FILE_NAME --not-index < $f
     fi
 
     if [[ $ARCH == "arm64" ]]; then
       archs[arm]=1
-      copy_to "$f" "$DISTRO/main/binary-arm64/"
-      # bundlearm=1
+      /usr/bin/docker exec -i repo-generator /generator.sh --distro $DISTRO --architecture arm --target $TARGET --name $FILE_NAME --not-index < $f
     fi
 
     if [[ $ARCH == "all" ]]; then
-      copy_to "$f" "$DISTRO/main/binary-amd64/"
-      copy_to "$f" "$DISTRO/main/binary-arm64/"
+      /usr/bin/docker exec -i repo-generator /generator.sh --distro $DISTRO --architecture amd --target $TARGET --name $FILE_NAME --not-index < $f
+      /usr/bin/docker exec -i repo-generator /generator.sh --distro $DISTRO --architecture arm --target $TARGET --name $FILE_NAME --not-index < $f
       archs[amd]=1
       archs[arm]=1
     fi
   fi
 done
 
-# no need to make bundle as we deploy one single package
-
-dev=
-if [ -n "$SUFFIX" ]; then
-  dev="-dev 1"
-fi
 
 for arch in "${!archs[@]}"; do
-  /usr/bin/docker exec repo-generator /generator.sh -distro "$DISTRO" -architecture "$arch" $dev
+  /usr/bin/docker exec -i repo-generator /generator.sh --distro $DISTRO --architecture "$arch" --target $TARGET --only-index
```

Comment:
WHat purpose of -i here? 

```
 -i, --interactive
                Keep STDIN open
                even if not attached
```

We should not run any interactive commands in non interactive envronment